android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/ParticipantManager.java

```diff
@@ -0,0 +1,111 @@
+package org.sagebionetworks.bridge.android.manager;
+
+import android.support.annotation.AnyThread;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import org.joda.time.LocalDate;
+import org.sagebionetworks.bridge.rest.api.ForConsentedUsersApi;
+import org.sagebionetworks.bridge.rest.model.StudyParticipant;
+import org.sagebionetworks.bridge.rest.model.UserSessionInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import rx.Completable;
+import rx.Single;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.sagebionetworks.bridge.android.util.retrofit.RxUtils.toBodySingle;
+
+/**
+ * Any authenticated user may use this class's methods. The user does not need to have consented to
+ * the study in order to manage their participant record.
+ * <p>
+ * TODO: offline syncing
+ */
+@AnyThread
```

Comment:
After reviewing the PlaintextSharedPreferencesDAO it does look like it is thread safe!  I figured you were storing some stuff in member variables, but you aren't.  Looks good to me!  Scratch this comment.